lib/tasks/docs.rake

```diff
@@ -507,7 +509,7 @@ namespace :docs do
   def lookbook_url(component)
     path = component.name.underscore.gsub("_component", "")
 
-    "https://primer.style/view-components/lookbook/inspect/#{path}/default/"
+    "https://primer.style/view-components/lookbook/inspect/#{path}/playground/"
```

Comment:
I realize I specifically requested this, but I don't know what I was thinking. We actually do want the default for docs 🤦‍♀️ 

```suggestion
    "https://primer.style/view-components/lookbook/inspect/#{path}/default/"
```